app/src/main/java/beepbeep/pixels/feed/FeedViewModel.kt

```diff
@@ -0,0 +1,17 @@
+package beepbeep.pixels.feed
+
+import beepbeep.pixels.feed.list.FeedListViewModel
+
+sealed class FeedAction
+
+class SetAction(val items: List<FeedListViewModel>) : FeedAction()
+object LoadAction : FeedAction()
+
+data class FeedViewModel(val isLoading: Boolean = false,
+                         val items: List<FeedListViewModel> = listOf())
+
+fun reduceWithAction(viewModel: FeedViewModel, action: FeedAction): FeedViewModel =
```

Comment:
should we namespace this thing? top-level functions can be a little scary after a while.

Reply:
I am open to it. What namespace you would like this to be inside?